libs/designsystem/tab-navigation/src/tab-navigation-item/tab-navigation-item.component.scss

```diff
@@ -0,0 +1,74 @@
+@use '@kirbydesign/core/src/scss/utils';
+
+$tab-item-text-max-width: 100px;
+$bottom-border-height: 1px;
+$border-color-standard: utils.get-color('medium');
+$border-color-selected: utils.get-color('dark');
+$divider-max-width-breakpoint: utils.$page-content-max-width + (utils.size('s') * 2);
+
+:host {
+  position: relative;
+  padding-bottom: $bottom-border-height * 2;
+
+  button[role='tab'] {
+    font-family: inherit;
+    background-color: utils.get-color('background-color');
+    color: utils.get-color('black');
+    padding: utils.size('s');
+    margin: 0;
+    font-size: utils.font-size('n');
+    font-weight: inherit;
+    line-height: utils.line-height('m');
+    cursor: pointer;
+    box-sizing: border-box; // Ensure border is not added to button height
+    display: inline-flex;
+    gap: utils.size('xxxs');
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    vertical-align: middle;
+    white-space: nowrap;
+    user-select: none;
+    text-decoration: none;
+    border: none;
+    outline: none;
```

Comment:
Consider placing all the relevant styles (I just marked some of them) for resetting the button styles in a mixin with a fitting name and include it here instead, just for this block to be a bit easier to understand.